magic-links/.env

```diff
@@ -1,23 +1,15 @@
-# required: true
-# format: sid
-# description: Your Twilio Account SID. Can be found in the Console.
-# link: https://www.twilio.com/console
-TWILIO_ACCOUNT_SID=
-
-# required: true
-# format: secret
-# description: Your Twilio Auth Token. Can be found in the Console.
-# link: https://www.twilio.com/console
-TWILIO_AUTH_TOKEN=
-
 # description: SID of your Twilio Verify Service
 # format: sid
 # link: https://www.twilio.com/console/verify/services
 # required: true
 VERIFY_SERVICE_SID=
 
-# description: URL used by the email template to check verifications
+# description: base URL linked in the email template to check verifications
 # format: url
 # required: true
-# default: http://localhost:3000/verify.html
-CALLBACK_URL=
\ No newline at end of file
+DOMAIN=http://localhost:3000
```

Comment:
If you swap that for `DOMAIN_NAME` it will be autopopulated by Functions with the deployed function base URL or with localhost:3000 if you run locally.

Reply:
oh neat this is super useful!